packages/vector_graphics/lib/vector_graphics.dart

```diff
@@ -114,7 +113,7 @@ class VectorGraphic extends StatefulWidget {
   /// This value does not apply to the widgets created by a [placeholderBuilder].
   /// Using this value is more efficient than wrapping this widget in an
   /// [Opacity] or [FadeTransition].
-  final double opacity;
+  final Animation<double>? opacity;
```

Comment:
See the docs on https://api.flutter.dev/flutter/widgets/Image/opacity.html

We should mention about using `AlwaysStoppedAnimation` to provide a fixed value.

Reply:
Updated the docs to mention AlwaysStoppedAnimation